src/lib-voyager.test.ui.tsx

```diff
@@ -93,11 +93,7 @@ describe('lib-voyager', () => {
           setTimeout(() => {
             try {
               const fieldList = document.querySelectorAll('.field-list__field-list-item');
-              const fields = Array.prototype.map.call(fieldList, (d: Node) => d.textContent);
-
-              expect(fields).toContain(' q1');
-              expect(fields).toContain(' q2');
-
+              expect(fieldList.length).toEqual(0);
```

Comment:
It seems like the description of this test doesn't really what the test does. 
(It tests two things -- maybe it "initializes with empty data and can be updated with cusomized data"?)